app/scripts/modules/core/domain/ISecurityGroup.ts

```diff
@@ -1,5 +1,24 @@
+import {ILoadBalancer} from './loadBalancer';
+import {ServerGroup} from './serverGroup';
+
+interface IUsages {
+  loadBalancers: ILoadBalancer[];
+  serverGroups: ServerGroup[];
```

Comment:
The `usages` really don't fit the interface: `usages.loadBalancers` just contains the `name` field; `usages.serverGroups` contains `name`, `isDisabled`, and `region`. Let's create new interfaces here instead of diluting the existing interfaces just to suit this particular scenario.